src/app/widgets/user-avatar/user-avatar.component.html

```diff
@@ -0,0 +1,5 @@
+<img class="user-assign-avatar"
+  [src]="imgSrc"
+  placement="left"
```

Comment:
placement should be generic. This should not be hardcoded in the component. Take placement as input.